### ISSUE TITLE
bug(mat-select): Talkback cannot navigate long dropdowns

### ISSUE THREAD

**Issue by shanti-razr** [angular/components#32628]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

When navigating through a mat-select element using Talkback on an Android mobile device, dropdowns with many options cause the page to not maintain focus and result in major jittering and page UI issues. This is not the case with a list of small options. This may be partially a mat-option issue as well.

### Reproduction

StackBlitz link: 
Steps to reproduce:
1. view page with Talkback on Android
2. attempt to move through the list (note the jittering and incorrectly moving content)

### Expected Behavior

Swiping left and right should provide consistent navigation without large UI jumps and page jittering. The element that is being focused should be indicated to the user.

### Actual Behavior

Swipping left and right with standard mobile screen reader navigation on long dropdowns results in UI reloading errors and focus issues. Occasionally, the first or last few elements will function normally but issues occur at the midpoints of these lists.

Looking at other UIs that have a similar element the Multiple Search Selection "state" field [here](https://semantic-ui.com/modules/dropdown.html) seems very good, UI focus is maintained and there are no large reloading jumps when navigating. Though this is an arbatray dropdown example, and I am sure there are other ones that could be identified.

### Environment

- Angular: 21.0.6
- CDK/Material: 20.2.9
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Android

**Comment by shanti-razr** [comment]
video recording, does not work on any version but here is an example on the most recent version. Going up seems to be fine but going down causes UI issues.

https://github.com/user-attachments/assets/4995ccce-9046-47d0-91c7-07b638428add

**Comment by shanti-razr** [comment]
This is a copy of a ticket that was previously opened and has not been resolved: https://github.com/angular/components/issues/32385

Though it was metioned as a potential fix, it seems to not be fixed by: https://github.com/angular/components/pull/32363
(unless this has not been included in the NEXT version on angular docs)

**Comment by crisbeto** [comment]
The fix should be in next.material.angular.io, but as you said, it might not have helped.
